Vagrantfile

```diff
@@ -280,6 +280,10 @@ EOF
           kubectl apply -f /vagrant/addon/traefik-ingress/
         fi
 
+        echo "Configure Kubectl to autocomplete"
+        source <(kubectl completion bash) # setup autocomplete in bash into the current shell, bash-completion package should be installed first.
+        echo "source <(kubectl completion bash)" >> ~/.bashrc # add autocomplete permanently to your bash shell.
```

Comment:
Not everyone uses bash, look at me, I am using zsh. But the CentoOS virtual machine uses bash on default.